### PR TITLE
FINDING-7696 Adding new trade project editor SVGs

### PR DESCRIPTION
**Pull request by ascagnel**
cc @twhid @hellatan 

Adding the new trade project editor SVGs.

### CODE REVIEW

File: src/gear-filled.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (nameofname) [inline src/gear-filled.svg]: clean these guys up to look more like the other SVGs : 
- kill the xml tags and comments
- the svg tag only needs the attributes version, class, viewBox, xmlns, xmlns:xlink, I usually remove the x/y attributes, definitely remove the ID, the style tags and xml:space. Honestly I don't quite know what they all do, but I would rather have these new ones be exactly consistent with the other ones, and if we find that we need those attributes we can add to all at once. 
-- here is an example from the arrow down SVG file : 
```html
<svg version="1.1" class="dibsvg-svg dibsvg-arrow-down" viewBox="0 -8.5 32 32" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment (ascagnel) [inline src/gear-filled.svg]: Should I add a dibs-vg specific class to each file (eg `class="dibsvg-svg dibsvg-gear-filled`)?

Comment (nameofname) [inline src/gear-filled.svg]: yes

**Comment by nameofname** [comment]
Looking better, still lacking dibsvg-svg classes.

**Comment by ascagnel** [comment]
Added classes, but I still need the new SVGs with paths.

**Comment by nameofname** [comment]
@ascagnel reading your last comment I am thinking I should wait on merging this? It looks ready to me, just waiting on your word.

**Comment by ascagnel** [comment]
@nameofname wait on merging this.  I haven't received the updated SVGs from the design team yet.

**Comment by ascagnel** [comment]
@nameofname updated with the latest SVGs, everything looks good now
screen shot 2016-05-18 at 8 56 14 am

File: src/gear-filled.svg [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```
Comment (nameofname) [inline src/gear-filled.svg]: Use the class dibsvg-svg then we are ready to go. Example I grabbed from another SVG file : 

```html
```

Comment (ascagnel) [inline src/gear-filled.svg]: Fixed.

**Comment by ascagnel** [comment]
Hold off a bit, just got a revision

**Comment by ascagnel** [comment]
Should be good to go now.

**Comment by nameofname** [comment]
@ascagnel merged, versioned, published.

**Pull request merged**